patches/minecraft/net/minecraft/block/OreBlock.java.patch

```diff
@@ -1,19 +1,21 @@
 --- a/net/minecraft/block/OreBlock.java
 +++ b/net/minecraft/block/OreBlock.java
-@@ -31,12 +31,10 @@
+@@ -31,7 +31,7 @@
  
     public void func_220062_a(BlockState p_220062_1_, ServerWorld p_220062_2_, BlockPos p_220062_3_, ItemStack p_220062_4_) {
        super.func_220062_a(p_220062_1_, p_220062_2_, p_220062_3_, p_220062_4_);
 -      if (EnchantmentHelper.func_77506_a(Enchantments.field_185306_r, p_220062_4_) == 0) {
--         int i = this.func_220281_a(p_220062_2_.field_73012_v);
--         if (i > 0) {
--            this.func_180637_b(p_220062_2_, p_220062_3_, i);
--         }
--      }
-+   }
++      if (false) {
```

Comment:
This looks more complex as a patch vs patch.
But the original patch is just deleting the entire code block
The new one is making it in a `if (false)` dead code block
The original patch is better in the sense that it will fail if code changes in that patch and we need to address it.
'large' patches are not a issue. COMPLEX patches are an issue.
'Just delete these lines' is stupid simple.
'lets rewrite and retab these 15 different functions, that uses 50 different variables just because I wanted to add one if statement above and my auto formatter changed it` is not

Reply:
thanks for clarifying